Description: fix note alignment
 It is architecture defined whether .align means .palign or .balign.  Use
 .balign to make that explicit.
Origin: upstream, https://code.qt.io/cgit/qt/qtbase.git/commit?id=47a4e6a9497b068a
Last-Update: 2026-05-05

--- a/src/corelib/global/minimum-linux.S
+++ b/src/corelib/global/minimum-linux.S
@@ -62,7 +62,7 @@
 
     .section    ".note.GNU-stack", "", progbits
     .section    ".note.ABI-tag", "a", note
-    .align      4       /* we have 32-bit data */
+    .balign     4       /* we have 32-bit data */
 
 /*  * For the format of the note section's contents, see Elf32_Nhdr / Elf64_Nhdr */
     .long       .Lnameend-.Lname        /* n_namesz */
